src/kernel/page.c

```diff
@@ -447,28 +444,28 @@ static boolean map_level(u64 *table_ptr, int level, range v, u64 *p, u64 flags,
             page_init_debug_u64(pte);
             page_init_debug("\n");
             table_ptr[i] = pte;
-            if (invalidate)
-                page_invalidate(fe, v.start | vmask);
```

Comment:
OK, I reinserted this page_invalidate() call, now using a null flush_entry argument, and modified the page_invalidate() function so that it just does a local cpu invalidation if the flush entry argument is null.